### DISCUSSION TITLE
PAZ processing with EZ-InSAR

### DISCUSSION THREAD

**Discussion by AllendePrieto** [alexisInSAR/EZ-InSAR#60] [Help]
Hi Alexis!
I'm trying to use the software using images captured by the PAZ satellite with two polarizations (HH and VV).  The SLC list generation is failing because the library only supports one polarization.  I've attached screenhost for reference.  Can you help me? Is there a version available that supports multiple polarizations?
Thanks!
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/131383377/7c12cdfc-ab20-4d7c-97d6-564368629ffc)
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/131383377/bb234f7d-62df-4f8c-b3e3-e2866d2dc2bd)

**Comment by alexisInSAR** [comment]
Hi, 

I need to solve this issue. Can you send me the list of names (name and directory structures) to implement the dual pol. images in EZ-InSAR? 

Kind regards, 

Alexis

**Reply by AllendePrieto** [reply]
Yes of course!
I attach a document with screenshots and the corresponding *.xml file.

Thank you!

[PAZ1_SAR__SSC______SL_D_SRA_20230104T062824_20230104T062825_XML.pdf](https://github.com/alexisInSAR/EZ-InSAR/files/12716609/PAZ1_SAR__SSC______SL_D_SRA_20230104T062824_20230104T062825_XML.pdf)
[Contenidos_PAZ_satellite.pdf](https://github.com/alexisInSAR/EZ-InSAR/files/12716612/Contenidos_PAZ_satellite.pdf)

**Reply by alexisInSAR** [reply]
Okay. I think I have the necessary documents to implement polarisation selection. 

Unfortunately, it will not be possible to process the HH and VV images at the same time, because I have to modify the entire workflow of ISCE... 

I will fix that this week. 

Kind regards,

Alexis

**Reply by AllendePrieto** [reply]
Ok great!!
Thank you so much!
Regards

**Comment by alexisInSAR** [comment]
Hi, 

The version 2.1.0 is available. I have implemented the management of dual polarisation for TSX and PAZ. I decided to create symbolic links to the desired polarisation. 

Please, do test it and let me know if it works (or not). 

Kind regards, 

Alexis

**Reply by AllendePrieto** [reply]
Hi Alex!
I have tried it, but it returns the following error:
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/131383377/769ee3f4-91cf-4410-8587-6b6dd9076d48)

**Reply by alexisInSAR** [reply]
You need to download the version 2.1.0. 

Kind regards, 

Alexis

**Reply by AllendePrieto** [reply]
I downloaded it, but the startup file reads:

![image](https://github.com/alexisInSAR/EZ-InSAR/assets/131383377/8b926f95-d2ab-46d4-a172-30e80289008e)

name is not updated
I'm doing something wrong

**Reply by alexisInSAR** [reply]
For the startup file, I forgot to change the version number. But on your previous screenshot, the version is 2.0.3 instead of 2.1.0. 

In addition, the createlistSLC.m script is not updated. 

WARNING: the version 2.1.0 is not the main branch. 

Kind regards, 

Alexis

**Reply by AllendePrieto** [reply]
I'll try again

**Comment by alexisInSAR** [comment]
Now, the default branch is the version 2.1.0.

**Reply by AllendePrieto** [reply]
Now it works!
Thank you!!

**Comment by AllendePrieto** [comment]
I have managed to start the processing. I have the reference date, but when I start the pre-processing with ISCE, I have this problem:

![image](https://github.com/alexisInSAR/EZ-InSAR/assets/131383377/f05a754d-a3cc-43ab-91bb-c9536d266e5f)
